edx_rest_framework_extensions/__init__.py

```diff
@@ -1,3 +1,3 @@
 """ edx Django REST Framework extensions. """
 
-__version__ = '10.5.0'  # pragma: no cover
+__version__ = '11.0.0'  # pragma: no cover
```

Comment:
Minor version bump will suffice.
```suggestion
__version__ = '10.6.0'  # pragma: no cover
```